src/Nethermind/Nethermind.Merge.Plugin/Data/ExecutionPayload.cs

```diff
@@ -183,12 +179,12 @@ public virtual bool TryGetBlock([NotNullWhen(true)] out Block? block, UInt256? t
     /// Decodes and returns an array of <see cref="Transaction"/> from <see cref="Transactions"/>.
     /// </summary>
     /// <returns>An RLP-decoded array of <see cref="Transaction"/>.</returns>
-    public Transaction[] GetTransactions()
+    public TransactionDecodingResult GetTransactions()
```

Comment:
May be worth renaming it `TryGetTransactions` - for consistency with `TryGetBlock` - all usages are already affected?